pkg/activator/handler/handler.go

```diff
@@ -226,8 +227,16 @@ func (a *activationHandler) proxyRequest(w http.ResponseWriter, r *http.Request,
 	network.RewriteHostIn(r)
 	recorder := pkghttp.NewResponseRecorder(w, http.StatusOK)
 	proxy := httputil.NewSingleHostReverseProxy(target)
-	proxy.Transport = &ochttp.Transport{
-		Base: a.transport,
+	config := activatorconfig.FromContext(r.Context())
+	isTraceEnabled := false
+	if config != nil {
```

Comment:
Should this ever be nil?

Reply:
well with the change above it might, but I agree that change above is not really needed.